packages/babel-core/src/transformation/plugin.js

```diff
@@ -77,6 +77,10 @@ export default class Plugin extends Store {
 
     this.maybeInherit(loc);
 
+    if (this.raw._c) {
```

Comment:
We are also throwing the original message below in `for (let key in this.raw) {` - if we do that we wouldn't need that message or need to substitute it?